.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- Mark items with [x] where applicable -->
+#### Have the results of the proposed changes been tested?
+- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
+- [ ] I generally don't use the affected packages but briefly tested this PR
+- [ ] I did not test this PR
```

Comment:
Remove this option. Being forced to leave the test options blank without testing might cause some people to do a bit more testing before submitting PRs.